packages/components/src/components/notification/notification.ts

```diff
@@ -0,0 +1,344 @@
+    }
+
+    this.open = true;
+    return waitForEvent(this, 'sd-after-show');
+  }
+
+  /** Hides the notification */
+  async hide() {
+    if (!this.open) {
+      return undefined;
+    }
+
+    this.open = false;
+    return waitForEvent(this, 'sd-after-hide');
+  }
+
+  /**
+   * Displays the notification as a toast notification. This will move the notification out of its position in the DOM and, when
+   * dismissed, it will be removed from the DOM completely. By storing a reference to the notification, you can reuse it by
+   * calling this method again. The returned promise will resolve after the notification is hidden.
+   */
+  async toast() {
+    return new Promise<void>(resolve => {
+      const toastStack = this.toastStack === 'bottom-center' ? toastStackBottomCenter : toastStackDefault;
+
+      if (toastStack.parentElement === null) {
+        document.body.append(toastStack);
+      }
+
+      toastStack.appendChild(this);
+
+      // Wait for the toast stack to render
+      requestAnimationFrame(() => {
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions -- force a reflow for the initial transition
+        this.clientWidth;
+        this.show();
+      });
+
+      this.addEventListener(
+        'sd-after-hide',
+        () => {
+          toastStack.removeChild(this);
+          resolve();
+
+          // Remove the toast stack from the DOM when there are no more alerts
+          if (toastStack.querySelector('sd-notification') === null) {
+            toastStack.remove();
+          }
+        },
+        { once: true }
+      );
+    });
+  }
+
+  render() {
+    return html`
+      <div
+        part="base"
+        class=${cx('w-full overflow-hidden flex items-stretch relative m-2')}
+        role="alert"
+        id="notification"
+        aria-hidden=${this.open ? 'false' : 'true'}
+        @mouseenter=${this.onHover}
+        @mouseleave=${this.onHoverEnd}
+      >
+        <slot
+          name="icon"
+          part="icon"
+          class=${cx(
+            'min-w-min flex items-center px-3 justify-center',
+            this.variant === 'info' && 'bg-info',
+            this.variant === 'success' && 'bg-success',
+            this.variant === 'warning' && 'bg-warning',
+            this.variant === 'error' && 'bg-error'
+          )}
+        >
+          <sd-icon
+            name=${this.variant === 'info'
+              ? 'info'
+              : this.variant === 'success'
+              ? 'success'
+              : this.variant === 'warning'
+              ? 'warning'
+              : this.variant === 'error'
+              ? 'error'
+              : ''}
+            library="system"
+            class="h-6 w-6 text-white"
+          ></sd-icon>
+        </slot>
+
+        <div
+          part="content"
+          class=${cx(
+            'h-full w-full gap-2 flex items-center justify-stretch bg-white',
+            'border-solid border-[1px] border-l-0 border-neutral-400',
```

Comment:
Why do you separate the classes at that point?
doen't it work like: 
`'h-full w-full gap-2 flex items-center justify-stretch bg-white border-solid border-[1px] border-l-0 border-neutral-400',`

And I somehow not understand how the border-[1px] is working 🤔

Reply:
@yoezlem It was more of a logical break to make it easier to read, with all the border styles are on one line and the positioning (mostly) styles are in another. I can put them into one line if you prefer that.

And the square brackets just allow me to put custom values in tailwind. There isn't a default value for a border width of 1px (minimum is 2 pixels with`border-2`)

Reply:
Thx for the clarification!
I will resolve this conversation, no need to change it! :)